src/utils/triggerDownload.ts

```diff
@@ -0,0 +1,9 @@
+export function triggerDownload(url: string, filename: string) {
```

Comment:
please use element ref's to call `click` methods inside the react component instead